src/components/flat-table/flat-table.style.ts

```diff
@@ -364,6 +364,14 @@ StyledFlatTableWrapper.defaultProps = {
 const StyledFlatTableFooter = styled.div<
   Pick<FlatTableProps, "hasStickyFooter">
 >`
+  & > ${StyledPagerContainer} {
+    border-top: none;
+    border-top-width: 0;
+    border-top-style: none;
+    border-top-left-radius: var(--borderRadius000);
```

Comment:
**comment** (non-blocking): I know we're supposed to use design tokens where possible as opposed to hard-coded pixel values - but I'm not sure about using a token here to say "remove the border radius". Will we ever want this to be a value other than `0px`? More to the point, will the DS?

I'm not saying to change this necessarily, particularly if we use the `borderRadius000` token elsewhere - but I'd rather be clear and put just `0px` rather than make readers of the code look up a token value, since what we actually want in this case is to have no radius in the top corners (or at least I thought so)... Worth thinking about I think - but happy for you to decide either way.